src/main/java/edu/cornell/kfs/sys/CUKFSConstants.java

```diff
@@ -177,6 +177,8 @@
 
     public static final String LOCATION_SERVICE_BEAN_NAME = "locationService-fin";
 
+    public static final String LEGACY_PERMIT_NAMESPACE = "PERMIT";
```

Comment:
Groups with the "PERMIT" namespace are related to our Active Directory group customization, which is on the Cynergy side and which Cynergy/Cynapps/GSC rely upon, but which we don't use on the KFS side. I could change the constant name to "PERMIT_NAMESPACE" if needed.

It would also be good to clean up the PERMIT-namespaced groups on the KFS side at some point, so I'll create a user story for that effort.

Reply:
Thank you for explaining this. Since we don't use this on the KFS side, can this be removed form the CUKFSConstants? Or it can only be removed when we cleanup the PERMIT-namespaced groups on the KFS side?

Reply:
There were some areas in our KFS code that were referring to the "PERMIT" namespace, but only for a few key features (such as preventing modifications to such groups). The constant containing that namespace was originally only available in cornell_customizations, so I added it here and adjusted the related code accordingly to allow things to compile. I've created KFSPTS-21620 to perform further cleanup of PERMIT group references on the KFS side.

Reply:
Thank you.